pkgs/os-specific/linux/rdma-core/default.nix

```diff
@@ -39,7 +38,8 @@ in stdenv.mkDerivation {
   postFixup = ''
     for pls in $out/bin/{ibfindnodesusing.pl,ibidsverify.pl}; do
       echo "wrapping $pls"
-      wrapProgram $pls --prefix PERL5LIB : "$out/${perl.libPrefix}"
+      substituteInPlace $pls --replace \
+        "${perl}/bin/perl" "${perl}/bin/perl -I $out/${perl.libPrefix}"
```

Comment:
Looking at the output, it seems to work. However, shouldn't the modified Perl environment be in `buildInputs`, such that the `fixupPhase` can automatically fix the shebang?

Reply:
Ideally yes, but I can't add `$out/${perl.libPrefix}` using perl.withPackages. This is one of the exceptional cases I mentioned.